src/datachain/lib/file.py

```diff
@@ -248,6 +262,10 @@ def read_text(self):
         with self.open(mode="r") as stream:
             return stream.read()
 
+    def stream(self) -> BytesIO:
+        """Returns file contents as BytesIO stream."""
+        return BytesIO(self.read())
```

Comment:
hmm, looks weird a bit? 

is `self.open` gives a stream?

otherwise this method is not streaming - I think it will read the whole file and just wraps it? it is then a confusing name for this

Reply:
🤔 You're right. I was confused with `Any` type.